### DISCUSSION TITLE
Improve API Rules UI

### DISCUSSION THREAD

**Discussion by StevenStavrakis** [pocketbase/pocketbase#1149] [Q&A]
Currently, the API rules UI looks like this:
<img width="689" alt="image" src="https://user-images.githubusercontent.com/35231833/204817853-2e23f001-e6da-4fca-a6ff-139b2a11b779.png">

The green unlock symbol makes it look like the element is already unlocked. An easy solution would be to swap the icons so that when locked, it is locked and green, and when unlocked, the icon is unlocked and black or a warning color like orange.

**Comment by ganigeorgiev** [comment]
I'm not sure about the suggestion. The green color here is used mainly to indicate visually that the rule is "safe/secure".

Additionally, please note that the "padlock" is not just an icon, it is a button and represents the action that will be performed, aka. in this case "unlock the field".

I'll spend some time experimenting with the API rules design in the future since I'm also planning some larger changes to the UI in relation to https://github.com/pocketbase/pocketbase/issues/976.

**Reply by StevenStavrakis** [reply]
If possible, I think UI elements that are interactive and UI elements that convey information should remain separate. A button should always convey what it will do when it is clicked. Right now, simultaneously shows that it will unlock, which is "bad," but colors it green, which is "good."

Instead of showing that it is green, as in good, I think it would be better to show it as disabled and maybe use a warning icon for when it is unlocked.

I don't have time right now for something better, but consider the following:

<img width="550" alt="image" src="https://user-images.githubusercontent.com/35231833/204831705-c5131c26-3266-4a05-862c-7d4b6827be9e.png">
